README.md

```diff
@@ -161,6 +161,18 @@ JSON.stringify(img._corners)
 => "[{"lat":51.52,"lng":-0.1},{"lat":51.52,"lng":-0.14},{"lat":51.5,"lng":-0.1},{"lat":51.5,"lng":-0.14}]"
 ```
 
+### Adding "DOMStrings" for custom toolbars
+
+Custom "DOMStrings" **(i.e., strings of HTML)** that define the custom toolbars can easily be added by describing those in the `src/edit/tools/DistortableImage.Guides.js` file and referencing those in the `addToolbar` method inside the `src/edit/DistortableImage.Edit.js` file.
```

Comment:
This is confusing to me. Can we break out a section called "Keymapper" just for describing how to create a Keymapper" if one is wanted. Then can discuss using that for custom toolbars